plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -220,6 +220,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 			Rules: []rbacv1.PolicyRule{
 				// TODO add future selfsubjectrulesreview, project request APIs, project listing APIs
 				rbacv1helpers.NewRule("create").Groups(authorizationGroup).Resources("selfsubjectaccessreviews", "selfsubjectrulesreviews").RuleOrDie(),
+				rbacv1helpers.NewRule("create").Groups(authenticationGroup).Resources("selfsubjectattributesreviews").RuleOrDie(),
```

Comment:
this means that distros that wish to restrict access to this API, must disable API when it goes GA.  I think this is reasonable, but wanted to call it out explicitly for other reviewers.